### ISSUE TITLE
Adding contrast accessibility on landing page of keycloak webpage

### ISSUE THREAD

**Issue by nehachopra27** [keycloak/keycloak-web#354]
### Description

Currently keycloak web page has contrast issues for accesibility.

URL to see the errors: https://wave.webaim.org/report#/https://www.keycloak.org/
<img width="1437" alt="Screenshot 2022-11-02 at 14 18 17" src="https://user-images.githubusercontent.com/11557840/200013548-5d3a88e8-6ae4-4c57-b5ce-1d8219d891e3.png">

### Motivation

Accessibility should be improved for differently abled users
